tools/python/onnx_test_data_utils.py

```diff
@@ -0,0 +1,166 @@
+import argparse
+import glob
+import os
+import sys
+
+import numpy as np
+
+from onnx import numpy_helper
+from onnx import mapping
+from onnx import TensorProto
+
+
+def dump_pb_file(filename):
+    """Dump the data from a pb file containing a Tensor."""
+
+    tensor = TensorProto()
+    with open(filename, 'rb') as f:
+        tensor.ParseFromString(f.read())
+
+    print("Name: {}".format(tensor.name))
+
+    np_array = numpy_helper.to_array(tensor)
+    print("Shape: {}".format(np_array.shape))
+    print(np_array)
+
+
+def dump_pb(dir_or_filename):
```

Comment:
Since there is function for dump_pb_file, I suggest to change this function to dump file under a directory. User shall choose which function to use.

Reply:
Not quite following. If action==dump_pb and the user provides a directory name it dumps all the pb files in the directory. If the user provides a filename (fully qualified or relative to the current directory) it will dump just that file.

e.g. I can provide a path like 'onnx\backend\test\data\node\test_abs\test_data_set_0' and it will dump the input and output pb files. Or I could provide 'onnx\backend\test\data\node\test_abs\test_data_set_0\input_0.pb' to just dump a single file.

What scenario would benefit from changing how it currently works?

Reply:
It is fine for this tool. Previously, I thought that this API might be imported by other script.